OpenRA.Game/WorldUtils.cs

```diff
@@ -44,8 +44,8 @@ public static IEnumerable<Actor> FindActorsInCircle(this World world, WPos origi
 
 		public static bool HasVoices(this Actor a)
```

Comment:
It should be relatively simple to remove these three now, but this might be scope creeping to much.